### ISSUE TITLE
Seems to struggle with dark theme.

### ISSUE THREAD

**Issue by netlore** [tomboy-notes/tomboy-ng#198]
With a dark theme, the colours in various dialog boxes such as sync, and search seem to be problematic, see attached image.,
![Screenshot from 2020-08-06 21-57-08](https://user-images.githubusercontent.com/457759/89582459-98189880-d830-11ea-9d86-0ada0e591ef6.png)

**Comment by davidbannon** [comment]
Thanks netlore, its heavily dependent on the Operating System, you have not mentioned yours.

The very ugly (in your case) notifier is replaced with a more tasteful (smaller and more appropriate colour) one on linux systems that use the libnotifier. I believe all Linux systems are moving to use Libnotifier (?).  And most Linux systems pick up most of their colours from the OS but strangely not all.  I have manually over ridden a few of the ones that dont but not all.  I guess I have dealt with the ones in regular use and not worried about windows used in frequently. Yet.

The windows dark theme is very ill defined. Its only a registry variable that may, or may not be set.  This then up to the app to apply suitable colours based on that setting. Again, I have done that to some parts of tomboy-ng but not in any way all.  The upstream Lazarus people have indicated that they won't address it holistically until Microsoft stop changing how its done. Late last year, it was changing with every Win10 update !

I am not sure what the situation is on the Mac, my only Mac testbed runs Sierra and Apple did not allow a dark theme until High Sierra from memory.

Can you tell me what OS your are using ?  And, maybe to further chase it down, which theme ?   I have seen some quite varying differences between some themes, particularly ones added as "after market".

Davo

**Comment by davidbannon** [comment]
Hmm, I realised that the form you quote uses TStringGrid, a component I have changed, where possible, because its overly complicated and does not pickup the system colours. So, that fixes the big white block to the left.

The panel that shows the text "All Done" shows with a black background on the three dark themes on my system, similarly, the white block or unused space to the right of the buttons. ??

Davo

**Comment by netlore** [comment]
Sorry, yes, I should have been more specific... this is your GTK package, running under Gnome 3.36.2 on Ubuntu 20.04, mostly vanilla install, bar the dark theme and some gnome extensions which do not affect the theme.

**Comment by davidbannon** [comment]
Yeah, and part of the problem was, and I have no idea why, I had disabled libnotifier on Linux. And as to why I did not spot it during testing, I really don't understand !

I think you will prefer v0.30

Sigh.....
